WalletWasabi.Packager/Program.cs

```diff
@@ -44,10 +43,10 @@ public static class Program
 
 		public static string PackagerProjectDirectory = Path.GetFullPath(Path.Combine(AppContext.BaseDirectory, "..", "..", ".."));
 		public static string SolutionDirectory = Path.GetFullPath(Path.Combine(PackagerProjectDirectory, ".."));
-		public static string GuiProjectDirectory = Path.GetFullPath(Path.Combine(SolutionDirectory, "WalletWasabi.Gui"));
+		public static string FluentProjectDirectory = Path.GetFullPath(Path.Combine(SolutionDirectory, "WalletWasabi.Fluent.Desktop"));
```

Comment:
Use the naming pattern "Desktop" or "FluentDesktop" instead of "Fluent". (Consistently everywhere.)

Reply:
I prefer Desktop btw.

Reply:
ACK